infer/src/base/Utils.ml

```diff
@@ -221,7 +221,22 @@ let do_finally_swallow_timeout ~f ~finally =
 
 
 let with_file_in file ~f =
-  let ic = In_channel.create file in
+  let ic =
+    let open Unix in
```

Comment:
local open police says to please avoid local opens :)

Reply:
Done.